tests/backup_helper.go

```diff
@@ -1666,6 +1667,72 @@ func ShareBackup(backupName string, groupNames []string, userNames []string, acc
 
 }
 
+// ShareCluster provides access to the mentioned groups or/add users
+func ShareCluster(clusterName string, groupNames []string, userNames []string, ctx context1.Context) error {
+	backupDriver := Inst().Backup
+	userIDs := make([]string, 0)
+
+	clusterUid, err := backupDriver.GetClusterUID(ctx, BackupOrgID, clusterName)
```

Comment:
@sgajawada-px I'm passing clusterUid as an input parameter for both methods in my PR. Fetching clusterUid may cause issues if the context is admin and the admin has multiple clusters from users with the same name. If you are planning to rebase with my branch, it's better to pass clusterUid at the test case level.